core/serialization/i_serializer.js

```diff
@@ -0,0 +1,70 @@
+/**
```

Comment:
We could, but to conform to the [styleguide](https://google.github.io/styleguide/jsguide.html#naming-hierarchy) we'd have to rename it to `Blockly.interfaces.ISerializer`. Personally I think it makes more sense to keep all of the serialization stuff in the same module, but there's probably something I'm unaware of here haha.

Reply:
Yeah that is good point. I think this a place where we currently deviate from the style guide. I guess it depends on whether we plan on moving towards the pattern the style guide recommends in the future or not. I'm not entirely sure what the plan for interfaces is with the ongoing conversion. We can leave this and add it to the list of things to check with Chris or Rachel on. If we don't plan on changing anything this quarter my vote would be to place it in the interface directory and ignore that style guide rule. (Since that is what we are doing for the other interfaces.) Then when we decide if we want to update the interfaces we can change them all at the same time.